en/SUMMARY.md

```diff
@@ -103,6 +103,7 @@
   * [Advanced Controller Orientation](advanced_config/advanced_flight_controller_orientation_leveling.md)
   * [Full Parameter Reference](advanced_config/parameter_reference.md)
 * [Peripheral Hardware](peripherals/README.md)
+  * [Companion Computer Peripherals](peripherals/companion_computer_peripherals.md)
```

Comment:
This is now sane, but can you please move it to the bottom after `[Camera](peripherals/camera.md)` ? 
My logic is that companion computers are not the first peripheral people need. If we end up with a huge list then I'd move to alphabetic, and this would be the right place.